docs/INSTALLATION.md

```diff
@@ -34,57 +34,57 @@
 
 To install from the command line:
 
-```
+```bash
```

Comment:
for command line commands that are not actually in bash scripts, `console` might be a better choice